docs/topics/lombok.md

```diff
@@ -0,0 +1,137 @@
+[//]: # (title: Lombok compiler plugin)
+
+> The Lombok compiler plugin is [Experimental](components-stability.md). 
+> It may be dropped or changed at any time. Use it only for evaluation purposes. 
+> We would appreciate your feedback on it in [YouTrack](https://youtrack.jetbrains.com/issue/KT-7112).
+>
+{type="warning"}
+
+The Lombok compiler plugin allows the generation and use of Java's Lombok declarations within the same mixed Java/Kotlin module.
+If you generate these declarations in a Java module and use them in another Kotlin module, 
+then you don't need to use this plugin.
+
+[Lombok](https://projectlombok.org/) generates Java code itself, not the Kotlin Lombok compiler plugin. 
+Thus, you still need to configure Lombok as usual, when using this plugin. 
+Learn more about [how to make the plugin seeing Lombok's config](#defining-the-place-of-lombok-config).
+
+## Support of annotations
```

Comment:
My concern about this whole section in the previous PR was that it seems unusable:
-  It's "Support of annotations" (should be "for", by the way). OK, let's see which annotations do we support.
- No supported annotations here! WTF? OK, there is a link. Let's go check here. Fine, I've found them after leaving the Kotlin website. Will I return to continue reading?
- If I return (hardly) I learn about what is not supported. It's a thought that continues what I've read on GitHub. Why is it here?

Reply:
We can erase this title and about @Builder, so this stuff will be a part of the previous common section. But the external link will anyway stay even in this case.